From f4ba92a1f91b07069d1236ce47f4859071018436 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Tue, 4 Oct 2016 16:16:42 -0700
Subject: [PATCH] MANTA-2997 muskie build broken after TOOLS-1556 Reviewed by:
 Trent Mick <trentm@gmail.com> Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Approved by: Trent Mick <trentm@gmail.com>

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index 039e31a..92996ea 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
         "dtrace-provider": "0.2.8",
         "http-signature": "1.1.0",
         "keep-alive-agent": "0.0.1",
-        "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#708745c",
+        "keyapi": "git+ssh://git@github.com:joyent/keyapi.git#c30dd2710ad2175095dc0e96479686fa774b8063",
         "libmanta": "git+ssh://git@github.com:joyent/node-libmanta.git#master",
         "libuuid": "0.1.2",
         "lru-cache": "2.3.1",
-- 
2.21.0

